stressor/stress-test.mts

```diff
@@ -2,33 +2,40 @@ import * as http from "node:http";
 import ngrok from "ngrok";
 import { Octokit } from "@octokit/rest";
 import { diff } from "jest-diff";
+import test, { run } from "node:test";
+import wrap from "word-wrap";
+import pLimit from "p-limit";
 
 const DEBUG = process.env.DEBUG === "true" || process.env.DEBUG === "1";
+const limitWorkflows = pLimit(8);
 
 const testPlans = [
-  // "tests/menu-button-actions-active-descendant",
-  // "tests/alert",
-  // "tests/horizontal-slider",
-  // "tests/command-button",
+  "tests/menu-button-actions-active-descendant",
+  "tests/alert",
+  "tests/horizontal-slider",
+  "tests/command-button",
   "tests/disclosure-navigation",
-  // "tests/link-span-text",
-  // "tests/modal-dialog",
-  // "tests/menu-button-navigation",
-  // "tests/radiogroup-aria-activedescendant",
-  // "tests/toggle-button",
+  "tests/link-span-text",
+  "tests/modal-dialog",
+  "tests/menu-button-navigation",
+  "tests/radiogroup-aria-activedescendant",
+  "tests/toggle-button",
 ];
-const owner = "bocoup",
+const owner = "gnarf",
```

Comment:
```suggestion
const owner = "bocoup",
```

need to reset this before merging, though we might want to paramaterize this to the script arguments or process.env